src/NuGet.Core/NuGet.LibraryModel/LibraryRange.cs

```diff
@@ -52,31 +52,42 @@ public string ToLockFileDependencyGroupString()
         {
             var sb = new StringBuilder();
             sb.Append(Name);
-            sb.Append(" ");
 
-            if (VersionRange == null)
-            {
-                return sb.ToString();
-            }
-
-            var minVersion = VersionRange.MinVersion;
-            var maxVersion = VersionRange.MaxVersion;
-
-            sb.Append(">= ");
-
-            if (VersionRange.IsFloating)
-            {
-                sb.Append(VersionRange.Float.ToString());
-            }
-            else
-            {
-                sb.Append(minVersion.ToString());
-            }
-
-            if (maxVersion != null)
+            if (VersionRange != null)
             {
-                sb.Append(VersionRange.IsMaxInclusive ? "<= " : "< ");
-                sb.Append(maxVersion.Version.ToString());
+                if (VersionRange.HasLowerBound || VersionRange.HasUpperBound)
```

Comment:
It was to avoid the extra space, but that was moved in farther. I'll take this if out